codegen_glibmm/templates/proxy.cpp.templ

```diff
@@ -53,6 +53,54 @@ void {{ class_name_with_namespace }}::{{ method.camel_name }}_finish(
     {% endfor %}
 }
 
+{% if not method is templated %}
```

Comment:
Would be nice if someone has any ideas of how to reduce code duplication between async and sync versions here.

Reply:
I would like to get rid of the special handling for templated methods. IIRC, they are only used if one of the method arguments is a `v` (which is very rare) and even in that case it's just adding some sugar: we should just accept a `Glib::VariantBase` in that case and let the caller do that little amount of work to prepare it.

Reply:
OK, I can have a look and see if I can remove it before adding this commit to simplify things.

Do you think it is something that is relied on much?

Reply:
Hum, it also looks like it would break if there is more than a single `v` argument.

Reply:
Well, for sure programs using `v` in their arguments will break, but it's an easy fix for them, and doing this in the master branch is fine.

Reply:
Phew! That took more work than expected. The integration test was broken. See the two new first commits.

Anyway, nuked template stuff and rebased the *_sync() stuff on top of that.

Marking this as resolved.